modules/combined/test/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp.i

```diff
@@ -44,13 +44,20 @@
   [../]
 []
 
-[Modules/TensorMechanics/Master/All]
-  displacements = 'disp_x disp_y'
-  add_variables = true
-  strain = SMALL
-  incremental = true
-  eigenstrain_names = eigenstrain
-  generate_output = 'stress_xx stress_yy stress_zz stress_xy stress_yz stress_zx'
+[Modules]
+    [TensorMechanics]
+        [Master]
+            displacements = 'disp_x disp_y'
```

Comment:
why is that line here?

Reply:
Are you not supporting the previous compact syntax with `/`s ?

Reply:
It is supported, but there is a long story about getting bison tests to pass and dealing with quirks of documentation system using the old parser still, etc. - so this needs to be like this (until we get python parser switched/converted)

Reply:
I bet it is long. You are not changing the old parser, so why do you need to change inputs for something that _uses_ the old parser? And why do we care about BISON tests in the combined module tests? If this is just going to be another "Bla, bla, bla" answer then don't bother, just let my dubiety be noted.

Reply:
The documentation system reads this file using the old parser.  With this PR, MOOSE reads this file using the new parser.  So we needed to find a version of the input that fit in the intersection of what the old and new parsers consider correct.

Reply:
We care about the BISON input files that use the tensor mechanics master action so that we can successfully pull in the relevant snippets in the BISON documentation; without somesort of work around, the documentation system will pull in the entire input file when any block below the master action is specified in the documentation system.  @dschwen I'd be happy to demonstrate this problem for you